setup.cfg

```diff
@@ -0,0 +1,15 @@
+[metadata]
+name = octrees
+version = attr: octrees.VERSION
+description = a Python implementation of the octree data structure and supporting algorithms
+long_description = file: README.md
+classifiers =
+    License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)
```

Comment:
Ah, oops: looking at this diff again has reminded me that some of the license fields in the built Python package (tarfile) seemed incomplete, on inspection.

I'll fix those in a follow-up pull request.